scripts/polygon-eth/token_transfer.py

```diff
@@ -0,0 +1,316 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2023 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+from web3 import Web3
+from eth_account import Account
+from web3.middleware import geth_poa_middleware
+from ledgereth import get_accounts, create_transaction, sign_transaction
+from ledgereth.web3 import LedgerSignerMiddleware
+import requests
+import json
+import os
+import binascii
+import sys, getopt
+import time
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+ALCHEMY_API_KEY_ETH = os.getenv("ALCHEMY_API_KEY_ETH")
+ALCHEMY_API_KEY_POLYGON = os.getenv("ALCHEMY_API_KEY_POLYGON")
+ETHERSCAN_API_KEY = os.getenv("ETHERSCAN_API_KEY")
+POLYGONSCAN_API_KEY = os.getenv("POLYGONSCAN_API_KEY")
+PRIVATE_KEY = os.getenv("PRIVATE_KEY")
```

Comment:
Consider calling env like so;

```
os.environ["ALCHEMY_API_KEY_ETH"] 
```
The reason being is that `getenv` will return none of the key is not set

However trying to access the dictionary like;
```
os.environ["ALCHEMY_API_KEY_ETH"] 
```
Will force the application to throw if the correct vars are not set, which might be preferable, as you fail fast and early